src/components/utils/useRadioGroup.ts

```diff
@@ -28,14 +38,22 @@ export function useRadioGroup(props: ControlGroupProps) {
         [isControlled, onUpdate, onChange],
     );
 
+    const containerProps = {
+        role: 'radiogroup',
+        'aria-label': props['aria-label'],
+        'aria-labelledby': props['aria-labelledby'],
```

Comment:
This role also allows `aria-disabled` https://w3c.github.io/aria/#radiogroup

Reply:
Yeah... Currently the component doesn't implement disabled prop at all. What do you think if it will simply set `disabled` on option components in this case?

Reply:
Fixed